extra/mtls/certs/README.md

```diff
@@ -1,3 +1,24 @@
-- `/server` - example self-signed server cert
-- `tenant-ca` - example self-signed tenant certificate authority ( key pass: root)
-- `client` - example client certs, using different algorithms, signed by `tenant-ca`
+# Example certificates for the mTLS setup
+
+This folder contains the example certificates for the mTLS setup:
+
+* `/server` - example self-signed server cert
+* `tenant-ca` - example self-signed tenant certificate authority ( key pass: root)
+* `client` - example client certs, using different algorithms, signed by `tenant-ca`
+
+You can create your own certificates following the [Mutual TLS documentation](https://docs.mender.io/2.7/server-integration/mutual-tls-authentication).
```

Comment:
This link to documentation is tricky. It cannot point to hosted (we don't have this doc there) but pointing it to 2.7 will make it deprecated soon. I don't have a solution... :thinking:

Reply:
Same here, no idea on how to fix it.